recipes-ros/catkin/catkin-runtime_0.6.19.bb

```diff
@@ -26,6 +27,27 @@ do_install_append() {
     rm  ${D}${ros_bindir}/catkin_make
     rm -rf ${D}${ros_datadir}
     rm -rf ${D}${ros_libdir}/pkgconfig
+
+    # Manually add the marker file
+    touch ${D}${ros_prefix}/.catkin
+}
+
+# Append environment hook for SDK
+do_install_append_class-nativesdk() {
+    mkdir -p ${D}${SDKPATHNATIVE}/environment-setup.d
+
+    script=${D}${SDKPATHNATIVE}/environment-setup.d/catkin-runtime.sh
+    echo "export ROSDISTRO=${ROSDISTRO}" >> $script
+    echo "export PATH=\$PATH:\${OECORE_NATIVE_SYSROOT}/opt/ros/\${ROSDISTRO}/bin" >> $script
+    echo "export PYTHONPATH=\${OECORE_NATIVE_SYSROOT}/opt/ros/\${ROSDISTRO}/lib/python2.7/site-packages" >> $script
```

Comment:
Don't we also need?

    \${OECORE_TARGET_SYSROOT}/usr/lib/python2.7/site-packages

Reply:
I will add it if you find out that it is needed.

Reply:
This shouldn't be needed; if it is, it is an indication something is wrong as this should work out of box.